config/networks.json

```diff
@@ -15,7 +22,14 @@
     "nativeCurrency": "ETH",
```

Comment:
do we not need decimals here, too? What if there ever is a network that does not have 18 decimals for its native currency?

Reply:
This most likely won't happen and it's easy to add later if need be. We don't use any functionality in Viem that actually depends on the `nativeCurrency` setup at the moment either